pykechain/models/part.py

```diff
@@ -11,16 +11,30 @@ def __init__(self, json, **kwargs):
         super(Part, self).__init__(json, **kwargs)
 
         self.category = json.get('category')
+        if 'parent' in json and json.get('parent') and 'id' in json.get('parent'):
```

Comment:
@wilcoschoneveld, if the json['parent'] exists in json BUT is 'None' we cannot query it for a subsequent dictionary. That is why there is an extra `json.get('parent') is not None` in the check.

    self.parent_id = json['parent'].get('id') if 'parent' in json and json.get('parent') else None

this works and all test pass in 35, will push and see if travis plays nice as well (in 27?). Nice compact code